go.mod

```diff
@@ -127,7 +128,7 @@ require (
 	github.com/charithe/durationcheck v0.0.10 // indirect
 	github.com/chavacava/garif v0.0.0-20230227094218-b8c73b2037b8 // indirect
 	github.com/coinbase/rosetta-sdk-go v0.7.9 // indirect
-	github.com/confio/ics23/go v0.9.0 // indirect
+	github.com/confio/ics23/go v0.9.1 // indirect
```

Comment:
Shouldn't this now be the one from cosmos and not confio?

I can't find the original confio repo to confirm tags

https://github.com/cosmos/ics23/blob/31ac3c774cefd3a2e55047c616773bf2ffead9d9/go/go.mod#L1